video/csputils.c

```diff
@@ -769,21 +747,12 @@ void mp_get_csp_matrix(struct mp_csp_params *params, struct mp_cmat *m)
         *m = (struct mp_cmat){{{0, 0, 1}, {1, 0, 0}, {0, 1, 0}}};
         break;
     }
+    case MP_CSP_XYZ:
```

Comment:
This change will break XYZ playback in "dumb" renderers like `vo_x11`.

Reply:
Update this PR, mostly inline with discussion in libplacebo PR. This one mostly fixes software conversion and vo_gpu while at it.